IslandLanding/IslandLanding/ViewModel/GameViewModel.cs

```diff
@@ -41,29 +116,35 @@ private void BackCommandExcute(object obj)
     private void ReadyCommandExcute(object obj)
     {
       IsStarting = true;
-      CalculateTimeRemaining();
+      // store time .now in varaible 
+      // when press launch compare time.now with varaible 
+      // then sub the 2 times 
+      //CalculateTimeRemaining();
+      Preferences.Set("levelnumber", LevelNumber);
+      StartedTime = DateTime.Now;
     }
 
     public void CalculateTimeRemaining()
     {
 
 
-      DateTime timeExpired = DateTime.Now.AddSeconds(8);
-      Xamarin.Forms.Device.StartTimer(new TimeSpan(0, 0, 1), () =>
-      {
-        var timespan = timeExpired - DateTime.Now;
-        if (timespan.TotalSeconds > 0)
-        {
-          Seconds = "in " + timespan.Seconds.ToString() + " s";
-          return true;
-        }
-        else
-        {
-          
-          Seconds = "0:0:0";
-          return false;
-        }
-      });
+      //DateTime timeExpired = DateTime.Now.AddSeconds(8);
```

Comment:
If you want to keep this, please add a Todo so if anyone has to pick this work, they know why it's commented out @amiraelsayedbadwy

Reply:
ok

Reply:
What's up with this code?